addons/binding/org.openhab.binding.homie/src/test/resources/simulateHomieMQTT.js

```diff
@@ -0,0 +1,75 @@
+var basetopic="homie/";
```

Comment:
Always hard to decide whether this should be part of the repo and in what form, I'm doubting.